index.js

```diff
@@ -4,29 +4,54 @@ const prog = require('caporal');
 const pkg = require('./package');
 const { downloadRepo } = require('./lib/repository');
 const { getLocalDependencies } = require('./lib/local');
-const { FROM, ERROR, OK } = require('./lib/constants');
+const Resolver = require('./lib/resolver');
+const { FROM, ERROR, DOWNLOADED, VALID, INVALID } = require('./lib/constants');
 
 function init(repo, logger) {
     downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+        files.forEach(file => logger.info(DOWNLOADED, `${file} ${FROM} ${repo}`));
     }).catch(err => logger.error(ERROR, err));
 }
 
 function initLocal(logger) {
     getLocalDependencies().then((deps) => {
+        if (!deps.length) {
+            logger.error(ERROR, 'No local dependencies defined. Please define a repository.');
```

Comment:
Could also `throw` and let the parent `catch` handle it. Redefining same handler.